### PR TITLE
Migrate common

### PR DESCRIPTION
**Pull request by bjw-s**
**Description of the change**

Move common library to separate repo

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 blank_issues_enabled: true
 contact_links:
 - name: Ask a question
-  url: https://github.com/k8s-at-home/charts/discussions
+  url: https://github.com/k8s-at-home/library-charts/discussions
```
Comment (onedr0p) [inline .github/ISSUE_TEMPLATE/config.yml:6]: What do you think about only having discussions enabled on that repo? sucks we can't have a org level discussion board.

File: .github/workflows/support.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
           to be a support request. Please use our support channels
           to get help with our Helm charts.
           - [Discord](https://discord.gg/sTMX7Vh)
-          - [GitHub Discussions](https://github.com/k8s-at-home/charts/discussions)
+          - [GitHub Discussions](https://github.com/k8s-at-home/library-charts/discussions)
```
Comment (onedr0p) [inline .github/workflows/support.yaml:23]: see other comment on discussion board

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+	"recommendations": [
+		"ms-vscode-remote.remote-containers"
+	]
+}
```
Comment (onedr0p) [inline .vscode/extensions.json:5]: new lines :P

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: .github/workflows/support.yaml (hunk 2 above)
Comment (bjw-s) [reply .github/workflows/support.yaml]: Fixed :)

File: .vscode/extensions.json (hunk 3 above)
Comment (bjw-s) [reply .vscode/extensions.json]: Fixed :)

**Pull request merged**